src/pages/ApiParticulier.js

```diff
@@ -82,24 +95,63 @@ const DonneesDescription = () => (
   </section>
 );
 
+const donneesDisponibles = [
+    {
+      "name": "dgfip_avis_imposition",
+      "humanName": "DGFIP - Avis d'imposition"
+    },
+    {
+      "name": "dgfip_adresse",
+      "humanName": "DGFIP - Adresse"
+    },
+    {
+      "name": "cnaf_quotient_familial",
+      "humanName": "CNAF - Quotient familial"
+    },
+    {
+      "name": "cnaf_attestation_droits",
+      "humanName": "CNAF - Attestation de droits"
+    }
+  ];
+
+// Le demandeur valide les modalités d'utilisation
+
 const CguDescription = () => <React.Fragment />;
+const cguLink = "https://particulier.api.gouv.fr/API_Particulier_modalites.pdf";
+
+// Le demandeur doit remplir des contenus supplémentaires
+const ContenusSupplementaires = () => <React.Fragment />;
 
 const ApiParticulier = ({
   match: {
     params: { enrollmentId },
   },
 }) => (
   <div className="documentation">
-    <Nav />
+    <Nav 
+      navLinksGeneral = {[
+        {id: "#demarche", text: "Démarche"},
```

Comment:
Please do not add a `#` before each id value but append it directly in the Nav component. The fact that this will be used as an anchor is an implementation detail that concern the Nav component, not the ApiParticulier component.